packages/cli/ete-tests/src/tests/fdr/__snapshots__/fdr.test.ts.snap

```diff
@@ -581,6 +610,44 @@ exports[`fdr {"name":"simple"} 1`] = `
                                 "type": "json",
                                 "value": "id-123"
                             }
+                        },
+                        {
+                            "path": "/",
+                            "pathParameters": {
+                                "movieId": "string"
```

Comment:
For OpenAPI when we do example generation, for strings we will copy the parameter name. Relevant [code](https://github.com/fern-api/fern/blob/9f1b2844bc4cdebc6e25e0bdcabf6acff94a4d12/packages/cli/openapi-parser/src/schema/examples/ExampleTypeFactory.ts#L487-L494)

Reply:
Maybe you can even use some of it